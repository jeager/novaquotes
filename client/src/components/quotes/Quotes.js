import React from 'react'
import { QuoteCard } from './quoteCard'
import './Quotes.css'

class Quotes extends React.Component {
  constructor (props) {
    super(props)
    this.renderQuotes = this.renderQuotes.bind(this)
  }

  renderQuotes () {
    return this.props.quotes.map(quote => {
      return <QuoteCard key={quote.id} quote={quote}/>
    })
  }

  render () {
    return (
      <div className="quotes">
        {this.renderQuotes()}
      </div>
    )
  }
}

export default Quotes
