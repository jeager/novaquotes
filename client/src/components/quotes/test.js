import React from 'react'
import Quotes from './'
import { shallow } from 'enzyme'
import ShallowRenderer from 'react-test-renderer/shallow'

test('Should render correctly', () => {
  const renderer = new ShallowRenderer()
  const component = renderer.render(<Quotes/>)

  expect(component).toMatchSnapshot()
})