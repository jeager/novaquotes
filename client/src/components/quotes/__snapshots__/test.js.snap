// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<div
  className="quotes"
>
  
</div>
`;
